### DISCUSSION TITLE
how to make w, e command stop at "-", iskeyword not work for me.

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#3860] [Q&A] [answered]
# solved 

```lua
vim.api.nvim_create_augroup("css minus", {})
vim.api.nvim_clear_autocmds({ group = "css minus" })
vim.api.nvim_create_autocmd({"BufRead","BufNewFile", "BufWinEnter"}, {
  pattern = "*.css",
  callback = function()
    vim.cmd("set iskeyword-=-")
  end,
})

```

---
I want `w` `e` command stop at "-". 
I try the config

`vim.cmd("set iskeyword+=-")`. But not work for me. So. how to fix it?

**Comment by Old-Farmer** [comment, marked as answer]
Use `set iskeyword-=-` in viml.
You can also use autocmd or after/ftplugin and `setlocal` to set option per buffer because some ftplugins will override your global settings.
